src/main/java/com/investmenttracker/enterprise/dto/Investment.java

```diff
@@ -3,7 +3,7 @@
 import lombok.Data;
 
 public @Data
-class investment {
+class Investment {
```

Comment:
Per best practices class names should be capitalized. There are a lot of these so I'm only going to tag this one, as I refactored it throughout the whole code base.